apps/system/test/marionette/browser_private_url_cleared_test.js

```diff
@@ -34,9 +33,7 @@ marionette('Private Browser - URL Persistence', function() {
     // Use the home-screen search box to open up the system browser
     var url = server.url('sample.html');
     rocketbar.homescreenFocus();
-    search.triggerFirstRun(rocketbar);
-    rocketbar.enterText(url + '\uE006');
-    system.gotoBrowser(url);
```

Comment:
I'm not sure why we need to trigger a first rocketbar run.
At the other hand, system.goToBrowser switches the frame to that url, and I've seen plenty of those without actually never checking anything in the frame.

My impression is that those 2 were added (in several apps) in an attempt to mitigate the issue I'm fixing with this PR.